ci/pipelines/deploy-to-staging.yml

```diff
@@ -991,6 +1017,23 @@ jobs:
           format: oci
         trigger: true
         passed: [selfservice-pact-tag]
+      - get: pay-ci
+      - task: parse-staging-release-tag
+        file: pay-ci/ci/tasks/parse-staging-release-tag.yml
+        input_mapping:
+          ecr-repo: selfservice-ecr-registry-staging
+      - load_var: application_image_tag
+        file: selfservice-ecr-registry-staging/tag
+      - load_var: parse_staging_release_tag
+        file: parse-staging-release-tag/tag
+      - task: send-job-release-info-to-hosted-graphite
+        file: pay-ci/ci/tasks/send-job-release-info-to-hosted-graphite.yml
+        params:
+          BUILD_PIPELINE_NAME: deploy-to-staging
+          BUILD_JOB_NAME: push-selfservice-to-production-ecr
+          RELEASE_NUMBER: ((.:parse_staging_release_tag))
+          HOSTED_GRAPHITE_API_TOKEN: ((HOSTED_GRAPHITE_API_KEY))
+
```

Comment:
Let's remove the extra blank line here - while it doesn't affect the functionality of the pipeline, we normally only have gaps between jobs (so it's easier for devs to see which tasks are part of a single job when skimming down the long YAML file).